srcpkgs/libjpeg-turbo/template

```diff
@@ -1,9 +1,9 @@
 # Template file for 'libjpeg-turbo'
 pkgname=libjpeg-turbo
 version=2.1.5.1
-revision=1
+revision=2
 build_style=cmake
-configure_args="-DWITH_JPEG8=1 -DCMAKE_INSTALL_LIBDIR=/usr/lib"
+configure_args="-DWITH_JPEG8=1 -DCMAKE_INSTALL_LIBDIR:PATH=lib${XBPS_TARGET_WORDSIZE}"
```

Comment:
We can remove it entirely, it has been set in `cmake` build-style.

Reply:
Build-style doesn't set the type to `PATH`, which is actuay significant here. We could probably do this there.
Idk if there might be a potential fallout, but that should be unlikely.